exercises/practice/anagram/anagram.ua

```diff
@@ -0,0 +1,3 @@
+# Find all the anagrams of a word in an array of words
+# Anagrams ? Target Words
+FindAnagrams ← |2 ⊙(⍤ "Please implement LargestProduct")
```

Comment:
```suggestion
FindAnagrams ← |2 ⊙(⍤ "Please implement FindAnagrams")
```

Reply:
Ah, dang it. I copy the signature over but keep forgetting to change the function name. thanks Erik